dashboard/dashboard-front/src/components/HeaderBar.vue

```diff
@@ -0,0 +1,67 @@
+<script setup lang="ts">
+import { computed, type Ref } from 'vue'
+import { mdiAccountCircle, mdiLogout } from '@quasar/extras/mdi-v7'
+import { isAuthenticated } from '@/services/UserDataStore'
+import { envInfo } from '@/services/EnvironmentInfo'
+
+const emit = defineEmits(['toggleDrawer'])
+
+function toggleDrawer () {
+    emit('toggleDrawer', null)
+}
+
+const siteNamePrefix: Ref<string> = computed(() => {
+    if (!!envInfo.site_name) {
```

Comment:
Why double `!`?

Reply:
I've seen somewhere it's a way to convert a value to a boolean, but now I see it should work the same without it.

Reply:
That's true, but in this case, it will work the same